fs/mount/fs_mount.c

```diff
@@ -293,7 +293,7 @@ int nx_mount(FAR const char *source, FAR const char *target,
 
   /* Find the specified filesystem. Try the block driver filesystems first */
 
-  if (source != NULL &&
+  if (source != NULL && strlen(source) > 0 &&
```

Comment:
why need strlen

Reply:
When source is "" and without this, there are duplicated "mm_free: Freeing ..." log during my test.

Reply:
It's better than strlen